app/controllers/static_page_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPageController < ApplicationController
+  def home
+  end
+
+  def help
+  end
```

Comment:
```suggestion
  def help; end
```
đọc lại coding convention nào cu